features/player_stats.lua

```diff
@@ -263,6 +279,15 @@ local function rocket_launched(event)
     change_for_global(satellites_launched_name, 1)
 end
 
+local function capsule_used(event)
+    local item = event.item
+
+    if not item or not item.valid or not item.name == 'raw-fish' then
```

Comment:
`not item.name == 'raw-fish'` is a bit of a lua trap, I'm guessing you think it's `not (item.name == 'raw-fish')` but it's actually `(not item.name) == 'raw-fish'` which is always `false` as `(not item.name)` is always `false`. Luckily there is an easy solution.
```suggestion
    if not item or not item.valid or item.name ~= 'raw-fish' then
```